src/components/Breadcrumb/Breadcrumb-story.js

```diff
@@ -42,10 +41,72 @@ storiesOf('Breadcrumb', module)
       },
     }
   )
+  .add(
+    'no trailing slash',
+    () => (
+      <Breadcrumb {...props()} noTrailingSlash>
+        <BreadcrumbItem>
+          <a href="/#">Breadcrumb 1</a>
+        </BreadcrumbItem>
+        <BreadcrumbItem href="#">Breadcrumb 2</BreadcrumbItem>
+        <BreadcrumbItem href="#">Breadcrumb 3</BreadcrumbItem>
+      </Breadcrumb>
+    ),
+    {
+      info: {
+        text:
+          'You can choose not to render a trailing slash with the `noTrailingSlash` prop',
+      },
+    }
+  )
   .add('skeleton', () => <BreadcrumbSkeleton />, {
     info: {
       text: `
           Placeholder skeleton state to use when content is loading.
           `,
     },
   });
+
+if (FeatureFlags.componentsX) {
```

Comment:
I guess this breaks because of the way we are exporting our feature flags? `FeatureFlags` is undefined, so we need to directly import `componentsX`. other than that the changes look good to me

Reply:
Good call! Thanks @emyarod